pkg/apis/operator/v1alpha1/config_types.go

```diff
@@ -9,6 +9,8 @@ import (
 type ConfigSpec struct {
 	// namespace where OpenShift pipelines will be installed
 	TargetNamespace string `json:"targetNamespace"`
+	// list of addon names to be installed with base
+	AddOns []string  `json:"addons,omitempty"`
```

Comment:
did you forget to generate the `CRD`.yaml

Reply:
I uploaded it.